package.json

```diff
@@ -23,6 +23,8 @@
     "node-fetch": "^2.6.0",
     "socket.io": "^2.3.0",
     "stripe": "^7.1.0",
+    "stylelint": "^13.2.1",
+    "stylelint-no-unsupported-browser-features": "^4.0.0",
```

Comment:
do the various es5ish changes flow from including this package?

Reply:
naw, I just added to inspect our CSS compatibility

I made the ES5 changes based on my internal understanding of browser support and some testing in IE11

but it's a good reminder to use `eslint-plugin-compat`